python/ray/serve/tests/test_standalone.py

```diff
@@ -473,5 +475,38 @@ class A:
             address=address, namespace="test_namespace2", port=8001))
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
```

Comment:
not related to this PR but do we know why serve tests fail on windows ?

Reply:
filesystem related stuff, reading and writing to tempdir mostly